shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -206,9 +219,11 @@ @implementation FlutterViewController {
  * Performs initialization that's common between the different init paths.
  */
 static void CommonInit(FlutterViewController* controller) {
-  controller->_engine = [[FlutterEngine alloc] initWithName:@"io.flutter"
-                                                    project:controller->_project
-                                     allowHeadlessExecution:NO];
+  if (controller->_engine == nullptr) {
```

Comment:
Nit: I believe the prevailing style in the macOS code is `!foo` rather than `foo == nullptr` for pointer checks.

Reply:
I see about 38 comparisons to `nullptr` in the `.mm` files in the repo. It's harder to search for references to using the bang operator, so I'm not sure how many of those there are, but I was able to find >100, so I definitely agree.  I'll switch this instance (I didn't make any others), and maybe we can make the others conform in another PR.